docs/2.9.0/docs/canton/usermanual/upgrading.rst

```diff
@@ -554,10 +627,16 @@ Now, using this configuration object, we can trigger the hard sync domain connec
 .. snippet:: migrating_protocol
     .. success:: participant.repair.migrate_domain("olddomain", config)
 
-This command will register the new sync domain and re-associate the contracts tied to ``olddomain`` to
-the new sync domain.
+This command registers the new sync domain and re-associate the contracts tied to ``olddomain`` to
+the new sync domain. In addition some data specific to the old domain are automatically deleted.
+
+.. warning::
+
+    The command ``repair.migrate_domain`` features a ``force`` flag. If you intentionally set it to
+    ``true`` then you force a sync domain migration ignoring any pending transactions. Forcing a
+    migration may result in a ledger fork. Use this flag as a last resort only.
```

Comment:
What is the reason/purpose to advertise this here ?

Reply:
Over-documentation -> removed.